.mocharc.yml

```diff
@@ -1,9 +1,9 @@
 recursive: true
 timeout: 7000
-parallel: true
```

Comment:
Local testing hangover?

Reply:
@rrsmith This was deliberate. Ironically, I lost a lot of time from the fact that running in parallel hid some essential error outputs. Have reverted it back as this can be a separate discussion/PR, but I do feel it's a footgun to have it enabled by default.

Reply:
Happy for you to remove them rather than just comment them out - we tend to have them on other repos just to get quicker feedback from CI (that should then trigger re-testing locally). Not an issue with this one though so would make sense to make the local testing case the default 👍